docs/_articles/en/user-guide/metadata-operations.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Metadata Operations
+lang: en
+---
+
+## Source Diff
+The Source Diff command allows you to diff metadata types against your default org. This feature makes it easier to visualize the changes between files in your local project and the metadata in your org. You can look at diffs for an individual file or directory.
+
+### Usage
+Menu options `SFDX: Diff File Against Org` and `SFDX: Diff Folder Against Org` appear when you right-click a file or folder.  
+
+![Source Diff command](../../../images/source_diff.png)
+
+You can then view individual file diffs side by side. If a file doesn’t exist locally or in the Org, it will not show up in the list.
+
+## Retrieve Source
+
+Org Development model doesn’t automatically track changes to your org, so be sure to keep track of the changes you retrieve. Retrieving source from an org overwrites the local versions of the source files. When you retrieve source from an org, you could enable conflict detection between the org and the local metadata. See [Detect Conflicts](./en/user-guide/detect-conflicts).
```

Comment:
Conflict detection is deploy-only so this should prob be removed:

> When you retrieve source from an org, you could enable conflict detection between the org and the local metadata. See [Detect Conflicts](./en/user-guide/detect-conflicts).

Reply:
Done. Thanks for catching that.